File: 07-Jenkins/freestyle-python/topic-02/code/broken.py
"""Deliberately invalid Python file.

This file exists on purpose: it has a syntax error (missing colon). It is
used to prove that the "Python Syntax Check" Execute Shell step actually
catches bad code instead of always reporting success. Do not fix it - the
guide fixes it live in the terminal to turn a red build green.
"""


def broken_function(a, b)
    return a + b

File: 07-Jenkins/kode-kloud/topic-01/code/broken.php
<?php
// Deliberately invalid PHP - a missing closing brace. Used to prove the
// "Syntax Check" Execute Shell step actually fails the build instead of
// always reporting green. Do not fix it here - fix it live on the Jenkins
// agent / in your simple-project checkout, then push again.

function brokenFunction($a, $b)
{
    return $a + $b;
